cypress/integration/app_admin/app_admin.js

```diff
@@ -1,17 +1,20 @@
 import { Given, When, Then, Before, After, And } from 'cypress-cucumber-preprocessor/steps';
 import { ADMIN_USER, ADMIN_NAME } from '../../support/constants';
-import { createUser } from '../../support/users';
 
 Before(() => {
-  createUser(ADMIN_USER, ADMIN_NAME);
-  cy.visit('/');
+  cy.deleteUser(ADMIN_USER).then(() => {

```

Comment:
burde sikkert bruke samme navn for rollene her og i frontend:
https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/types/user.types.ts#L3

Reply:
Må uansett ta en ny runde med roller når rolleopplegget er klart.